File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingOutlook/WorkingWithOutlookMessageFiles/ParseOutlookMessageFile.php
<?php
namespace Aspose\Email\ProgrammingOutlook\WorkingWithOutlookMessageFiles;

use com\aspose\email\MapiMessage as MapiMessage;

class ParseOutlookMessageFile{

    public static function run($dataDir=null){

        $mapiMessage=new MapiMessage();
        $outlook_message_file = $mapiMessage->fromFile($dataDir . "Message.msg");

        # Display sender's name
        print "Sender Name : " . $outlook_message_file->getSenderName();

        #Display Subject
        print "Subject : " . $outlook_message_file->getSubject();

        # Display Body
        print "Body : " . $outlook_message_file->getBody();
    }
}
?>

File: Plugins/Aspose-Email-Java-for-Jython/asposeemail/ProgrammingOutlook/ParseOutlookMessageFile.py
from asposeemail import Settings
from com.aspose.email import MapiMessage

class ParseOutlookMessageFile:

    def __init__(self):
                        
        dataDir = Settings.dataDir + 'ProgrammingOutlook/WorkingWithOutlookMessageFiles/ParseOutlookMessageFile/'
        
        mapiMessage=MapiMessage()
        outlook_message_file = mapiMessage.fromFile(dataDir + "Message.msg")

        # Display sender's name
        print "Sender Name : " 
        print outlook_message_file.getSenderName()

        #Display Subject
        print "Subject : " 
        print outlook_message_file.getSubject()

        # Display Body
        print "Body : " 
        print outlook_message_file.getBody()
        
 
if __name__ == '__main__':        
    ParseOutlookMessageFile()